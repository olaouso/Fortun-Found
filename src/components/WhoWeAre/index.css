.whoWeAre {
  font-family: "Acme";
}
.targetText {
  display: flex;
  justify-content: center;
  align-items: center;
  padding-left: 3em;

  width: 100%;
  line-height: 170%;
  text-align: center;
  height: auto;
}

.targetHeader {
  font-size: 1.75em;
  color: #03c4a1;
}

.targetParagraph {
  padding: 1.8em;
  font-size: 1.25em;
  color: #726a95;
}

.whoWeAreImage {
  width: 70%;
  height: auto;
  padding: 1%;
  margin: 1%;
}

.wrapper {
  width: 100%;
  height: 100%;
  margin: 0 auto;
}
.innerVideo {
  position: relative;
  display: flex;
  align-items: center;
  justify-content: center;
  /* background: #03c4a1; */
  padding: 2%;
}

.whoWeAreTitle {
  display: flex;
  align-items: center;
  justify-content: center;
  font-family: "Acme";
  text-align: center;
  color: #590995;
  font-size: 2.5em;
  padding-top: 0.2em;
}
