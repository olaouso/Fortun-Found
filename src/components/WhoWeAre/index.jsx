import React from "react";
import { useTranslation } from "react-i18next";
import { Row, Col } from "react-bootstrap";
import target from "./images/target.jpg";
import "./index.css";

export default function WhoWeAre() {
  const { t } = useTranslation();
  return (
    <div fluid="md" className="whoWeAre">
      <Row className="targetText">
        <Col md={6}>
          <h2 className="targetHeader">{t("whoWeAre.header")}</h2>
          <p className="targetParagraph">{t("whoWeAre.paragraph")}</p>
        </Col>

        <Col md={6}>
          <img src={target} alt="Target" className="whoWeAreImage" />
        </Col>
      </Row>

      <Row>
        <Col>
          <h1 className="whoWeAreTitle">{t("whoWeAre.howItWorks")}</h1>
        </Col>
      </Row>

      <Row className="innerVideo">
        <iframe
          height="500em"
          width="800em"
          src="https://www.youtube.com/embed/9Fev-hRCR-8"
          frameborder="0"
          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
          allowfullscreen
          title="How It Works?"
        ></iframe>
      </Row>
    </div>
  );
}
