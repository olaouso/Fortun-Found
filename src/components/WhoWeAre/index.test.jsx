import React from "react";
import renderer from "react-test-renderer";
import WhoWeAre from "./index";

it("renders WhoWeAre correctly", () => {
  const tree = renderer.create(<WhoWeAre />).toJSON();
  expect(tree).toMatchSnapshot();
});
