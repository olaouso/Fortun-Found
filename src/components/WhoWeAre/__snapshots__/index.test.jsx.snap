// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders WhoWeAre correctly 1`] = `
<div
  className="whoWeAre"
  fluid="md"
>
  <div
    className="targetText row"
  >
    <div
      className="col-md-6"
    >
      <h2
        className="targetHeader"
      >
        whoWeAre.header
      </h2>
      <p
        className="targetParagraph"
      >
        whoWeAre.paragraph
      </p>
    </div>
    <div
      className="col-md-6"
    >
      <img
        alt="Target"
        className="whoWeAreImage"
        src="target.jpg"
      />
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <h1
        className="whoWeAreTitle"
      >
        whoWeAre.howItWorks
      </h1>
    </div>
  </div>
  <div
    className="innerVideo row"
  >
    <iframe
      allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
      allowfullscreen={true}
      frameborder="0"
      height="500em"
      src="https://www.youtube.com/embed/9Fev-hRCR-8"
      title="How It Works?"
      width="800em"
    />
  </div>
</div>
`;
